sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileClient.java

```diff
@@ -496,6 +499,57 @@ public FileReadResponse readWithResponse(OutputStream stream, FileRange range, D
         }, logger);
     }
 
+    /**
+     * Opens a file input stream to download the file.
+     * <p>
+     *
+     * @return An <code>InputStream</code> object that represents the stream to use for reading from the file.
+     * @throws DataLakeStorageException If a storage service error occurred.
+     */
+    public DataLakeFileInputStream openInputStream() {
+        return openInputStream(null);
+    }
+
+    /**
+     * Opens a file input stream to download the specified range of the file.
```

Comment:
May wish to document the default handlings used, such as if a read consistency isn't provided it'll use `ConsistentReadControl.ETAG`.